packages/hcloud/build.sh

```diff
@@ -0,0 +1,27 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/hetznercloud/cli
+TERMUX_PKG_DESCRIPTION="Hetzner Cloud command line client"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.30.1"
+HCLOUD_PKG_VERSION="v${TERMUX_PKG_VERSION}"
```

Comment:
Please drop this variable and just specify `v${TERMUX_PKG_VERSION}` in SRCURL and `LD_FLAGS`